client/src/app/components/MenuActions.tsx

```diff
@@ -17,23 +17,26 @@ export const MenuActions: React.FC<MenuActionsProps> = ({ actions }) => {
     <Dropdown
       isOpen={isOpen}
       onSelect={() => {
-        setIsOpen(!isOpen);
+        setIsOpen(false);
```

Comment:
This seems counterintuitive to me.

Reply:
when a selection is made, we want to close the dropdown

Reply:
the dropdown items have their own. i suppose this is part of being composable and having more flexibility. could make the dropdown item have an onclick with `() => setIsOpen(false); item.callback();`, but it's nice that it can be handled by the dropdown for all selections.

Reply:
+1 - After looking into this, seems that there are never any menuItems passed to the PageHeader component. I think this is a leftover component that just never got deleted.